### ISSUE TITLE
[Bug]: `GLIBC_2.32' not found (required by /management-state/bin/kontainer-engine-driver-lke)

### ISSUE THREAD

**Issue by rovaniemi** [linode/kontainer-engine-driver-lke#46]
### Expected Behavior

The upgrade should be successful from v0.0.6 to v0.0.8 on rancher 2.8.0. 

### Actual Behavior

Error during installation: 

```
/management-state/bin/kontainer-engine-driver-lke: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by /management-state/bin/kontainer-engine-driver-lke)
/management-state/bin/kontainer-engine-driver-lke: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by /management-state/bin/kontainer-engine-driver-lke)
```

Rancher was installed with docker

**Comment by lgarber-akamai** [comment]
@rovaniemi Thank you for the report!

It looks like our [release CI](https://github.com/linode/kontainer-engine-driver-lke/blob/main/scripts/build#L13-L27) is currently building binaries with dynamic linking (`CGO_ENABLED=1`) which can cause similar issues on certain distributions/images. 

This issue should be resolved in #45 and I'll let you know once it has been released 🙂

**Comment by rovaniemi** [comment]
It would be great if that fixes the problem. I found that downgrading go below 1.2 might fix that problem also, but `CGO_ENABLED=1` would be a much more elegant solution.

**Comment by lgarber-akamai** [comment]
@rovaniemi We just cut a new release for statically linked binaries. Could you validate that your issue is resolved using the following driver configuration?

Download URL: `https://github.com/linode/kontainer-engine-driver-lke/releases/download/v0.0.9/kontainer-engine-driver-lke-linux-amd64`
Checksum: `f489f3b354280f8a2859945de27c76b0a70a888976d4cebcb58a30fe161f4b97`

**Comment by rovaniemi** [comment]
@lgarber-akamai this issue has been resolved in version 0.0.9. Thank you!
